### PR TITLE
Titles project

### PR DESCRIPTION
**Pull request by mattgle**
## Summary
Titles  "Training maquetado"

## Trello Card
https://trello.com/c/ohUh7exV/21-page-titles
## Screenshots
captura de pantalla 2018-10-22 a la s 12 20 48

### CODE REVIEW

File: maquetado/styles.scss [hunk 1]
```diff
@@ -1,3 +1,5 @@
-.title {
-  color: red;
+body {
+  background-repeat: no-repeat;
```
Comment (henryzarza) [inline maquetado/styles.scss]: I think that this property isn't necessary delete it

File: maquetado/styles.scss [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+  background-size: 50%;
```
Comment (henryzarza) [inline maquetado/styles.scss]: The same here

File: maquetado/styles.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
...
+  font-family: 'Roboto';
```
Comment (henryzarza) [inline maquetado/styles.scss]: To use a font as fallback in case that the primary font can’t load

**Review by henryzarza — CHANGES_REQUESTED** [review]
Good job only a few comments and we ready!

**Review by henryzarza — APPROVED** [review]

**Review by SKOLZ — APPROVED** [review]

**Pull request merged**